### ISSUE TITLE
The wheels and source releases for 1.0 differ

### ISSUE THREAD

**Issue by mariusvniekerk** [sphinx-contrib/confluencebuilder#192]
The wheel release is broken because it contains a translator subpackage and a .py file causing import errors

```
Running Sphinx v1.8.5

Extension error:
Could not import extension sphinxcontrib.confluencebuilder (exception: cannot import name 'ConfluenceTranslator')
```

This is caused by there being two `translator` packages in `confluencebuilder` and it picks the wrong one.

Unpacking the two archives shows this.

```
unzip sphinxcontrib_confluencebuilder-1.0.0-py3-none-any.whl

Archive:  sphinxcontrib_confluencebuilder-1.0.0-py3-none-any.whl
  inflating: sphinxcontrib_confluencebuilder-1.0.0-py3.7-nspkg.pth
  inflating: sphinxcontrib/confluencebuilder/__init__.py
  inflating: sphinxcontrib/confluencebuilder/__main__.py
  inflating: sphinxcontrib/confluencebuilder/assets.py
  inflating: sphinxcontrib/confluencebuilder/builder.py
  inflating: sphinxcontrib/confluencebuilder/compat.py
  inflating: sphinxcontrib/confluencebuilder/config.py
  inflating: sphinxcontrib/confluencebuilder/exceptions.py
  inflating: sphinxcontrib/confluencebuilder/experimental.py
  inflating: sphinxcontrib/confluencebuilder/logger.py
  inflating: sphinxcontrib/confluencebuilder/publisher.py
  inflating: sphinxcontrib/confluencebuilder/rest.py
  inflating: sphinxcontrib/confluencebuilder/state.py
  inflating: sphinxcontrib/confluencebuilder/translator.py
  inflating: sphinxcontrib/confluencebuilder/util.py
  inflating: sphinxcontrib/confluencebuilder/writer.py
  inflating: sphinxcontrib/confluencebuilder/std/__init__.py
  inflating: sphinxcontrib/confluencebuilder/std/confluence.py
  inflating: sphinxcontrib/confluencebuilder/std/sphinx.py
  inflating: sphinxcontrib/confluencebuilder/translator/__init__.py
  inflating: sphinxcontrib/confluencebuilder/translator/shared.py
  inflating: sphinxcontrib/confluencebuilder/translator/wiki.py
  inflating: sphinxcontrib_confluencebuilder-1.0.0.dist-info/namespace_packages.txt
  inflating: sphinxcontrib_confluencebuilder-1.0.0.dist-info/top_level.txt
  inflating: sphinxcontrib_confluencebuilder-1.0.0.dist-info/WHEEL
  inflating: sphinxcontrib_confluencebuilder-1.0.0.dist-info/METADATA
  inflating: sphinxcontrib_confluencebuilder-1.0.0.dist-info/RECORD
```

```
tar xvf 'sphinxcontrib-confluencebuilder-1.0.0.tar.gz'
x sphinxcontrib-confluencebuilder-1.0.0/
x sphinxcontrib-confluencebuilder-1.0.0/CHANGES.rst
x sphinxcontrib-confluencebuilder-1.0.0/LICENSE
x sphinxcontrib-confluencebuilder-1.0.0/MANIFEST.in
x sphinxcontrib-confluencebuilder-1.0.0/PKG-INFO
x sphinxcontrib-confluencebuilder-1.0.0/README.rst
x sphinxcontrib-confluencebuilder-1.0.0/setup.cfg
x sphinxcontrib-confluencebuilder-1.0.0/setup.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/__init__.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/__init__.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/__main__.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/assets.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/builder.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/config.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/exceptions.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/logger.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/publisher.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/rest.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/state.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/std/
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/std/__init__.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/std/confluence.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/std/sphinx.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/translator.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/util.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib/confluencebuilder/writer.py
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib_confluencebuilder.egg-info/
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib_confluencebuilder.egg-info/PKG-INFO
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib_confluencebuilder.egg-info/SOURCES.txt
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib_confluencebuilder.egg-info/dependency_links.txt
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib_confluencebuilder.egg-info/namespace_packages.txt
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib_confluencebuilder.egg-info/not-zip-safe
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib_confluencebuilder.egg-info/requires.txt
x sphinxcontrib-confluencebuilder-1.0.0/sphinxcontrib_confluencebuilder.egg-info/top_level.txt
x sphinxcontrib-confluencebuilder-1.0.0/test/
x sphinxcontrib-confluencebuilder-1.0.0/test/test_main.py
x sphinxcontrib-confluencebuilder-1.0.0/test/test_validation.py
```

**Comment by jdknight** [comment]
Thanks for the report. Seems the WHL file has legacy implementation mixed with new implementation; although the sources file (from a quick glace) appears to have correct content. Will need to generate a new release package.

**Comment by jdknight** [comment]
I believe I have corrected this with a quick release of v1.1 with is primarily just a re-packaging of the implementation. A local check and post-PyPI package check seems to be a working state.

@mariusvniekerk, if you still see an issue with the package, please feel free to make another comment. Thanks again for bringing this up.

**Comment by jdknight** [comment]
Since no other reports have been made about the recent release package, assuming v1.1 is in a good enough state for the current extension's capabilities. Marking this issue as closed.
